src/game/f3dex3.c

```diff
@@ -0,0 +1,388 @@
+#include <ultra64.h>
+
+#ifdef F3DEX_GBI_3
+#include "game_init.h"
+#ifdef DEBUG_F3DEX3_PROFILER
+#include "buffers/buffers.h"
+#include "puppyprint.h"
+#include "fasttext.h"
+#endif
```

Comment:
Nit: Space these out a bit to make them easier to visually parse, and/or add comments to the `endif`s